### ISSUE TITLE
doctest fail in segmentation.rst

### ISSUE THREAD

**Issue by cdeil** [astropy/photutils#237]
Looks like there's a doctest fail in `segmentation.rst` now:
https://travis-ci.org/astropy/photutils/jobs/47222670#L2015

```
__________________________ [doctest] segmentation.rst __________________________
065 Now let's measure the properties of the detected sources defined in
066 the segmentation image with the minimum number of inputs to
067 `~photutils.segmentation.segment_properties`:
068 
069 .. doctest-requires:: scipy, skimage
070 
071     >>> from photutils import segment_properties, properties_table
072     >>> props = segment_properties(data, segm)
073     >>> tbl = properties_table(props)
074     >>> print(tbl)
Differences (unified diff with -expected +actual):
    @@ -1,4 +1,4 @@
    - id   xcentroid     ycentroid    ...       cxy             cyy
    -         pix           pix       ...     1 / pix2        1 / pix2
    + id   xcentroid     ycentroid    ...       cxy             cyy
    +         pix           pix       ...     1 / pix2        1 / pix2
     --- ------------- -------------- ... ---------------- --------------
       1 234.983156946 0.985867670057 ...  -0.290404010574  1.55891445615
    @@ -7,5 +7,15 @@
       4 364.729681583  10.9844940633 ...    -1.2288069393 0.635093403231
       5 258.031283251  11.9649001764 ...  0.0599262879469 0.808873726802
    +  6 289.449970606  22.3424368541 ... -0.0282787403518 0.682935600292
    +  7 379.302608654  28.6107587692 ...   0.480224234574 0.710834977986
    +  8 441.343847296  31.1280050515 ...  -0.199936731221 0.379194837779
    +  9 358.556758296  36.2830444775 ...    1.16295988183 0.541287182813
    + 10 470.743181082  38.4329861887 ...   0.412148319289 0.622839391948
     ...           ...            ... ...              ...            ...
    + 72 397.366766517  247.187105875 ...  -0.305421693111 0.643339266411
    + 73 446.519523896  248.042486152 ...   0.575554351179 0.729208936565
    + 74 355.641175372  252.120131162 ...   0.148969870492 0.423898855256
    + 75 98.0189554022  254.149269803 ...   0.232109296376 0.794200433096
    + 76 74.4134456659  259.832954108 ...   0.549943995677 0.637875329876
      77  82.553164741  267.507957132 ...  0.0959470832587 0.468289801445
      78 477.742087097  267.723968829 ... -0.0470970087203 0.520702684332
    @@ -13,2 +23,3 @@
      80  434.09234827  280.841150924 ...   -1.45089455818 0.815770166696
      81 434.091385011  288.922811107 ...  -0.258098406023 0.454297620264
    +Length = 81 rows

/home/travis/build/astropy/photutils/docs/photutils/segmentation.rst:74: DocTestFailure
```

@larrybradley Can you reproduce and fix this one?
If the table formatting changed we'll probably just have to skip this as long as we support Astropy 0.4?

**Comment by cdeil** [comment]
Looks like this is an intermittent fail ... the next build with the same config passes, although there were no changes to that build:
https://travis-ci.org/astropy/photutils/builds/47228838

!?

**Comment by larrybradley** [comment]
I can't reproduce this doctest fail.

**Comment by cdeil** [comment]
Closing this now ... if someone can reproduce it locally in the future please comment here and we'll re-open.

**Comment by larrybradley** [comment]
It just showed up again on `travis-ci`:
https://travis-ci.org/astropy/photutils/jobs/47286568

Very odd.

**Comment by larrybradley** [comment]
I truncated the long astropy table to included the first and last 5 rows.  From the error diff, it appears that the doctest truncates the table using the first and last **10** rows.  I have no idea though why the doctest  fails intermittently.

**Comment by cdeil** [comment]
I don't understand this either, but there was a recent change in table formatting:
https://github.com/astropy/astropy/pull/3239

Maybe we should skip this doctest and add the exact same example as a unit test and a comments linking the two? Or is there a better way to have the docs tested, but not rely on the exact output?

**Comment by larrybradley** [comment]
The test fails because astropy/astropy#3239 prints the table length at the end of the table if the table isn't completely shown (e.g. this line https://travis-ci.org/astropy/photutils/jobs/47286568#L2053).  There is not an option to turn off this behavior (I'll submit a PR for that).  The doctest will need to be skipped until we require `astropy >= 1.0` (implemented in #238).
